builds/e2e/longhaul.yaml

```diff
@@ -94,10 +94,10 @@ jobs:
           longHaul.desiredModulesToRestartCSV: '$(longHaul.desiredModulesToRestartCSV)'
           longHaul.restartIntervalInMins: '$(longHaul.restartIntervalInMins)'
           analyzer.consumerGroupId: 'longhaul_linux_amd64'
-          analyzer.logAnalytic.enabled: '$(analyzer.logAnalytic.enabled)'
-          analyzer.logAnalytic.workspaceId: '$(kvLogAnalyticWorkspaceId)'
-          analyzer.logAnalytic.sharedKey: '$(kvLogAnalyticSharedKey)'
-          analyzer.logAnalytic.logType: '$(analyzer.logAnalytic.logType)'
+          analyzer.logAnalytics.enabled: '$(analyzer.logAnalytics.enabled)'
+          analyzer.logAnalytics.logType: '$(analyzer.logAnalytics.logType)'
+          logAnalytics.workspaceId: '$(kvLogAnalyticWorkspaceId)'
+          logAnalytics.secret: '$(kvLogAnalyticSharedKey)'
```

Comment:
why you renamed it from analyzer.logAnalytic.sharedKey to logAnalytics.secret? did you discuss these changes with Bear? you may need to use different log type for different module result.

Reply:
why you remove analyzer prefix for logAnalytics.workspaceId and logAnalytic.sharedKey? are these 2 parameters going to reuse in another module?

Reply:
@philipktlin 
It looks like Andrew is introducing it to "Metric" module which apparently a different module than Analyzer. Do you think it would be alright to have those parameter log to the same `logType` as the Analyzer results? Currently I have logType be a type of a pipeline: `lhReleasePipeline`, `lhReleaseCandidatePipeline`, vice versa.

Reply:
@philipktlin Bear is correct. Another 1-2 modules are going to be added to the deployment which will use these same fields.

Reply:
The metrics collector will log to a different log type. Will need to look at source to know which one.

Reply:
The rename from sharedKey to secret is for consistency with venkat's logspout module, which I had planned to use. It seems though that now I might not need it if I can get the same functionality with azure functions.